openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/DatabaseRepository.java

```diff
@@ -116,6 +130,40 @@ private void populateService(Database database) {
     database.setServiceType(service.getServiceType());
   }
 
+  public Database addDatabaseProfilerConfig(UUID databaseId, DatabaseProfilerConfig databaseProfilerConfig) {
+    // Validate the request content
+    Database database = dao.findEntityById(databaseId);
+
+    if (databaseProfilerConfig.getProfileSampleType() != null && databaseProfilerConfig.getProfileSample() != null) {
+      EntityUtil.validateProfileSample(
+          databaseProfilerConfig.getProfileSampleType().toString(), databaseProfilerConfig.getProfileSample());
+    }
+
+    daoCollection
+        .entityExtensionDAO()
+        .insert(
```

Comment:
maybe silly question, is this actually an UPSERT?

Reply:
ummm not sure on this, replicated the logic that we already had for table profiler config [here](https://github.com/open-metadata/OpenMetadata/blob/bdfc9226d5d31db18cbb846c04fe4bea48021264/openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TableRepository.java#L320) 🤔

Reply:
ah cool yes it takes care of it https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java#L614 sorry should've looked at it before